238d18b2dc33500d51e77cdbad52c74a281d12de is the way to go with the dash shell, the default shell for `#!/bin/sh` on Ubuntu.
But maybe change this:
```patch
-echo "* Installing composer packages '$@'"
+echo "* Installing composer packages '$*'"
```
https://www.shellcheck.net/wiki/SC2145

--- a/scripts/create-release.sh
+++ b/scripts/create-release.sh
@@ -562,17 +562,19 @@ composer config platform.php "$PHP_REQ"
 composer update --no-interaction --no-dev
 
 # Parse the required versions from composer.json
-PACKAGES_VERSIONS=''
 PACKAGE_LIST='tecnickcom/tcpdf pragmarx/google2fa-qrcode bacon/bacon-qr-code code-lts/u2f-php-server web-auth/webauthn-lib thecodingmachine/safe'
 
-for PACKAGE in $PACKAGE_LIST
+set --
+
+for PACKAGES in $PACKAGE_LIST
 do
-    PKG_VERSION="$(get_composer_package_version "$PACKAGE")"
-    PACKAGES_VERSIONS="$PACKAGES_VERSIONS\n$PACKAGE:\"$PKG_VERSION\""
+    PKG_VERSION="$(get_composer_package_version "$PACKAGES")"
+    set -- "$@" "$PACKAGES:$PKG_VERSION"
 done
 
-echo "* Installing composer packages '$(echo "$PACKAGES_VERSIONS" | tr $'\n' ' ')'"
-IFS=$'\n' composer require --no-interaction --update-no-dev $PACKAGES_VERSIONS
+echo "* Installing composer packages '$*'"
+
+composer require --no-interaction --update-no-dev "$@"
 
 echo "* Running a security checkup"
 security_checkup

